esphome/components/sx1509/__init__.py

```diff
@@ -0,0 +1,77 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome import pins
+from esphome.components import i2c
+from esphome.const import CONF_ID, CONF_NUMBER, CONF_MODE, CONF_INVERTED
+
+CONF_ON_TIME = 'on_time'
+CONF_OFF_TIME = 'off_time'
+CONF_RISE_TIME = 'rise_time'
+CONF_FALL_TIME = 'fall_time'
+CONF_ON_INT = 'on_intensity'
+CONF_OFF_INT = 'off_intensity'
```

Comment:
```suggestion
CONF_OFF_INTENSITY = 'off_intensity'
```

Use the matching name for CONF_ constant. Same above.